doc/samples/readme-encode_linux.md

```diff
@@ -107,6 +107,7 @@ The following command-line switches are optional:
   | [-num_slice] | number of slices in each video frame. 0 by default. If num_slice equals zero, the encoder may choose any slice partitioning allowed by the codec standard.|
    |[-mss] |maximum slice size in bytes. Supported only with -hw and h264 codec. This option is not compatible with -num_slice option.|
   | [-mfs]|maximum frame size in bytes. Supported only with h264 and hevc codec for VBR mode.|
+  | [-BitrateLimit:<on,off>] | Turn this flag ON to set bitrate limitations imposed by the SDK encoder: if target bitrate is lower than suggested by the SDK encoder, it will be changed to suggested. The default value is OFF, i.e. bitrate is not limited.|
```

Comment:
Modify bitrate to be in the range imposed by the SDK encoder. Setting this flag off may lead to violation of HRD conformance. The default value is OFF, i.e. bitrate is not limited.|

Reply:
changed